### PR TITLE
Article Length Filter

### PR DESCRIPTION
**Pull request by mbouadeus**
Create filter bar component separate from browse page.
Separate different filters to different states so we can have more than one filter for each activity type [games, videos, articles].
Add article length filter UI: https://screenshot.googleplex.com/b0b611ba-a1b7-4bb7-a075-c51378f1e17c.png
Implement filtering by article length functionality.

### CODE REVIEW

**Review by trinitydonohugh — APPROVED** [review]
LGTM :)

**Review by pengxianzhe — APPROVED** [review]

File: src/main/app/src/constants/ActivitiesFilterBar.js [hunk 1]
```diff
@@ -1,5 +1,98 @@
...
+      case activityTypes.GAMES:
+        updateNumOfPlayers(value);
+        break;
+      case activityTypes.VIDEOS:
+        updateVideoType(value);
+        break;
+      case activityTypes.ARTICLES:
+        if (name === "ArticleType")
+          updateArticleType(value);
+        else
```
Comment (isaacwdavis) [inline src/main/app/src/constants/ActivitiesFilterBar.js]: nit: should this be else if (name === "ArticleLength")?

Also maybe add another console.log error in an else statement

**Review by isaacwdavis — APPROVED** [review]

File: src/main/app/src/constants/ActivitiesFilterBar.js (hunk 1 above)
Comment (mbouadeus) [reply src/main/app/src/constants/ActivitiesFilterBar.js]: Yeah, I think that would be best as well.

Comment (mbouadeus) [reply src/main/app/src/constants/ActivitiesFilterBar.js]: Fixed it

**Pull request merged**